### PR TITLE
Accounting guide provider

### PR DESCRIPTION
**Pull request by cthermolia-grnet**

### CODE REVIEW

File: website/docs/guides/provider.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+id: provider
+title: Manage providers
+sidebar_position: 1
+---
+### Manage Providers
```
Comment (ctriant) [inline website/docs/guides/provider.md:6]: Remove redundant heading

File: website/docs/guides/provider.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### Before you start 
+You can manage a provider assigned to a specific project.
+Firstly you need to be registered to the accounting-system and then you need to contact the administrator of the project , that this provider is associated, in order to assign you one or more roles, on the provider.
```
Comment (ctriant) [inline website/docs/guides/provider.md:10]: You can manage a provider assigned to a specific project.
Firstly, you need to be registered to the accounting-system and then you need to contact the administrator of the project, that this provider is associated with, to assign you one or more roles on the provider.

File: website/docs/guides/provider.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Once you gain permissions on the specific provider, you can act on it, according to the permissions. 
+
+### READ the provider
+<details>
+You can display the providers's details. Apply a request to the api. 
+<b> For more details , see <a href="https://argoeu.github.io/argo-accounting/docs/api/provider#get---fetch-an-existing-provider">here</a></b>
```
Comment (ctriant) [inline website/docs/guides/provider.md:16]: Space after comma

File: website/docs/guides/provider.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+</details>
+
+### ASSIGN installations to the provider
+<details>
+You can assign one or more installations to the installation. Apply a request to the api.
```
Comment (ctriant) [inline website/docs/guides/provider.md:21]: You can assign one or more installations to the provider?

File: website/docs/guides/provider.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+<b> For more details , see <a href="https://argoeu.github.io/argo-accounting/docs/api/installation#post---create-a-new-installation">here</a></b>
```
Comment (ctriant) [inline website/docs/guides/provider.md:22]: Remove space after comma

File: website/docs/guides/provider.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+</details>
+
+### COLLECT all installations assigned to the provider
+<details>
+You can collect and display the details of all the installations assigned to the provider. Apply a request to the api
+<b> For more details , see <a href="https://argoeu.github.io/argo-accounting/docs/api/installation#get-fetch-all-provider-installations">here</a></b>
```
Comment (ctriant) [inline website/docs/guides/provider.md:28]: Remove space after comma

File: website/docs/guides/provider.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+</details>
+
+### SEARCH installations
+<details>
+You can search for specific installation assigned to the provider, that matches one or more criteria. Apply a request to the api. You need to provide the search criteria in a specific <b><a href=""> syntax</a></b>. <b> For more details , see <a href="">here</a></b>
```
Comment (ctriant) [inline website/docs/guides/provider.md:33]: Remove space after comma

File: website/docs/guides/provider.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+</details>
+
+<b> Also, on the installations assigned to the provider, you are permitted to perform all the actions described at this <a href="../guides/installation"> section</a></b>
```
Comment (ctriant) [inline website/docs/guides/provider.md:36]: Maybe rephrase to: "You are permitted to perform all the actions described at ....  on the installations assigned to the provider."

File: website/docs/guides/installations.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+id: installation
+title: Manage Installations
+sidebar_position: 2
+---
+### Manage Installations
+
+### Before you start
+
+You can manage an installation assigned to a specific project and provider.
+Firstly you need to be registered to the accounting-system and then you need to contact the administrator of either the project or the provider, associated to the installation, in order to assign you one or more roles, on the installation.
```
Comment (ctriant) [inline website/docs/guides/installations.md:11]: **Firstly,** you need to be registered to the accounting-system and then you need to contact the administrator of either the project or the provider, associated **with** the installation, ~~in order~~ to assign you one or more roles, on the installation.

File: website/docs/guides/installations.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+Once you gain permissions on the specific installation, you can act on it, according to the permissions.
```
Comment (ctriant) [inline website/docs/guides/installations.md:12]: Once you gain permissions on the specific installation, you can act on it according to the permissions.

File: website/docs/guides/search_filter.md [hunk 11]
```diff
@@ -80,7 +80,7 @@ The syntax should be as:
 
 ```
 
-#### Example 2: Search on a combination of 2 fields, of the collection
+#### Example 2: Search on a combinations of 2 fields, of the collection
```
Comment (ctriant) [inline website/docs/guides/search_filter.md:83]: combination~~s~~

File: website/docs/guides/search_filter.md [hunk 12]
```diff
@@ -80,7 +80,7 @@ The syntax should be as:
...
 You search for metrics , that the start period is after 01-01-2022 AND the end period if before 01-02-2022. The syntax of the filter should be :
```
Comment (ctriant) [inline website/docs/guides/search_filter.md:84]: ~~if~~ is (?)

File: website/docs/guides/search_filter.md [hunk 13]
```diff
@@ -80,7 +80,7 @@ The syntax should be as:
 
 ```
 
-#### Example 2: Search on a combination of 2 fields, of the collection
+#### Example 2: Search on a combinations of 2 fields, of the collection
 You search for metrics , that the start period is after 01-01-2022 AND the end period if before 01-02-2022. The syntax of the filter should be : 
 
 At first 2 subqueries should be created to define the criterio on each field.
```
Comment (ctriant) [inline website/docs/guides/search_filter.md:86]: At first,

Comment (ctriant) [reply website/docs/guides/search_filter.md:86]: criterio~~n~~

File: website/docs/guides/search_filter.md [hunk 14]
```diff
@@ -135,8 +135,7 @@ Now these two queries should be combined in a filter as:
 
 ```
 
-
-#### Example 3: Search on a multiple fields, of the collection
+#### Example 3: Search on multiple fields, of the collection
 You search for metrics , that  start period is after 01-01-2022 and the end period if before 01-02-2022,  OR the value is greater than 1000. The syntax of the filter should be  as :
```
Comment (ctriant) [inline website/docs/guides/search_filter.md:139]: ~~if~~ is (?)

File: website/docs/guides/search_filter.md [hunk 15]
```diff
@@ -135,8 +135,7 @@ Now these two queries should be combined in a filter as:
 
 ```
 
-
-#### Example 3: Search on a multiple fields, of the collection
+#### Example 3: Search on multiple fields, of the collection
 You search for metrics , that  start period is after 01-01-2022 and the end period if before 01-02-2022,  OR the value is greater than 1000. The syntax of the filter should be  as : 
 
 The filter to search for   start period is after 01-01-2022 and the end period if before 01-02-2022, if defined as in the Example 2.
```
Comment (ctriant) [inline website/docs/guides/search_filter.md:141]: ~~if~~ is (?)

File: website/docs/guides/providers.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+<b> For more details,how to syntax the request, see <a href="https://argoeu.github.io/argo-accounting/docs/api/provider#get---fetch-an-existing-provider">here</a></b>
+</details>
+
+### ASSIGN installations to the provider
+<details>
+You can assign one or more installations to the provider. Apply a request to the api.
+<b> For more details,how to syntax the request, see <a href="https://argoeu.github.io/argo-accounting/docs/api/installation#post---create-a-new-installation">here</a></b>
+</details>
+
+### COLLECT all installations assigned to the provider
```
Comment (themiszamani) [inline website/docs/guides/providers.md:30]: ## Fetch all the installations assigned to the Provider

File: website/docs/guides/providers.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+<details>
+You can collect and display the details of all the installations assigned to the provider. Apply a request to the api
+<b> For more details,how to syntax the request, see <a href="https://argoeu.github.io/argo-accounting/docs/api/installation#get-fetch-all-provider-installations">here</a></b>
+</details>
+
+### SEARCH installations
```
Comment (themiszamani) [inline website/docs/guides/providers.md:36]: Search for Provider's Installations

File: website/docs/guides/providers.md [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+<details>
+You can search for specific installation assigned to the provider, that matches one or more criteria. Apply a request to the api. You need to provide the search criteria in a specific <b><a href="https://argoeu.github.io/argo-accounting/docs/guides/search-filter"> syntax</a></b>. <b> For more details,how to syntax the request, see <a href="https://argoeu.github.io/argo-accounting/docs/api/installation#post---search-for-installations">here</a></b>
```
Comment (themiszamani) [inline website/docs/guides/providers.md:38]: Could you please add specific criteria of the search. 
Not how to .. but what i can search

**Review by themiszamani — APPROVED** [review]
